packages/solo/src/web.js

```diff
@@ -10,6 +10,10 @@ import morgan from 'morgan';
 
 import { getAccessToken } from '@agoric/access-token';
 
+const textDecoder = new TextDecoder();
+
+const maximumBundleSize = 1024 * 1024 * 128; // 128MB
```

Comment:
Arbitrary. Open to suggestions for a less arbitrary mechanism or a less arbitrary magic number.